packages/generator/src/v2.0/generateMetadata.ts

```diff
@@ -31,7 +32,7 @@ export async function generateOntologyMetadataFile(
       export const OntologyMetadata = {
         ontologyRid: "${ontology.ontology.rid}",
         ontologyApiName: "${ontology.ontology.apiName}",
-        userAgent: "foundry-typescript-osdk/2.0.0",
+        userAgent: "${userAgent}",
```

Comment:
this tripped me up on self-review, the whole thing is actually inside backticks so this is valid syntax.